componentTemplates/workflows/testPublish.yml

```diff
@@ -0,0 +1,46 @@
+name: Test and publish
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the main branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
```

Comment:
Again, this is not static. In the generator, we use an app variable that is set to whatever the currently supported version is. 

https://github.com/AlaskaAirlines/WC-Generator/blob/3a9b8996afb6509e92d58869ad024e33bbc5a813/package.json#L20-L22

https://github.com/AlaskaAirlines/WC-Generator/blob/3a9b8996afb6509e92d58869ad024e33bbc5a813/bin/generate.js#L259